### PR TITLE
Add CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by whymarrh**
See [_Creating a default community health file - GitHub Help_](https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file)

This PR adds an **organization-wide** default `CODE_OF_CONDUCT.md` file. This file defines standards for how to engage in our community and who to contact with complaints.

### CODE REVIEW

**Comment by whymarrh** [comment]
This file is the [Contrubutor Covenant](https://www.contributor-covenant.org/version/2/0/code_of_conduct/) but I'm open to alternatives. @danfinlay had proposed copying the Django CoC in https://github.com/MetaMask/Code-of-Conduct/pull/1, which is an option.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (whymarrh) [inline CODE_OF_CONDUCT.md:64]: We'd need an appropriate email for this

Comment (danfinlay) [reply CODE_OF_CONDUCT.md:64]: I've contacted IT and requested `metamask-conduct@consensys.net` as an email & google group we can maintain.

Comment (danfinlay) [reply CODE_OF_CONDUCT.md:64]: We now have this address, as well as `conduct@metamask.io`. So we can insert that as the contact.

**Pull request merged**